lib/dai-plugin-mcd/test/ManagedCdp.spec.js

```diff
@@ -128,6 +173,15 @@ describe.each([
       myDai: startingDaiBalance.plus(1),
       myGem: startingGemBalance.minus(1)
     });
+    const price = await cdp._cdpType.getPrice();
+    await expectUtilValues(cdp, {
+      val: price,
+      ratio: price.toNumber(),
+      isSafe: true,
+      min: true,
```

Comment:
it's more consistent to pass in values for `min` and `avail` here rather than do the calculations inside the helper function

Reply:
I did this because they are dependent calculations based on price which is already submitted as val. They are dependent on that val and can't really be tested separately. How might you suggest that we could pass in a value instead

Reply:
hmm, now i see what you're saying. maybe we don't even need to toggle them independently from val, and we can just do all three checks when `val` is passed in.